var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -85,6 +94,9 @@ def cmake_args(self):
         if '~shared' in self.spec:
             options.append('-DBUILD_SHARED_LIBS:BOOL=OFF')
 
+        if '+static' in self.spec:
+            options.append('-DGMX_PREFER_STATIC_LIB:BOOL=ON')
+
```

Comment:
Avoid relying on defaults in upstream build, and make the recipe more clear and compact, by doing stuff like this:
```
options = [
    '-DGMX_PREFER_STATIC_LIB:BOOL=%s' % 'ON' if '+static' in spec else 'OFF',
    ...
]
```

Reply:
or even more concisely:

```python
'-DGMX_PREFER_STATIC_LIB:BOOL=%s' % ('+static' in spec)
```

CMake understands `True` and `False` (which the `bool` above will stringify to).